httpx/config.py

```diff
@@ -1,5 +1,6 @@
 import ssl
 import typing
+from collections.abc import Iterable
```

Comment:
Why `collections.abc.Iterable` instead of `typing.Iterable`?
https://docs.python.org/3/library/typing.html#typing.Iterable

Reply:
I didn't think to use types like this :) thanks